eagle-core/eagle-data-process/eagle-stream-process-api/src/test/java/org/apache/eagle/datastream/TestExecutionEnvironmentJava.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.eagle.datastream;
```

Comment:
http://creadur.apache.org/rat/apache-rat-plugin/rat-mojo.html  I think this might help.